src/wallet/sqlite.h

```diff
@@ -105,7 +105,7 @@ class SQLiteDatabase : public WalletDatabase
 private:
     const bool m_mock{false};
 
-    const std::string m_dir_path;
+    const fs::path m_dir_path;
 
     const std::string m_file_path;
```

Comment:
nit, maybe while you are touching this anyways:

```suggestion
    const fs::path m_file_path;
```

Reply:
Yes. I don't see any benefit of doing it now.